exercises/isogram/.meta/generator/isogram_case.rb

```diff
@@ -18,7 +18,7 @@ def failure_message
   end
 
   def reason
-    "'#{input}' #{is_or_not} an isogram"
+    "'\#{string}' #{is_or_not} an isogram"
```

Comment:
I think "input" is reasonable here as a name, as perhaps the input may not be a string, and the statement would still be true, it is or not an isogram.  The generic is informative here, in my opinion.